.travis.yml

```diff
@@ -27,7 +27,7 @@ script:
   - test ! -d ./html/modules/custom || find -L ./html/modules/custom -iregex '.*\.\(php\|module\|inc\|install\)$' -print0 | xargs -0 -n 1 -P 4 php -l
 
   # Build local image.
-  - make
```

Comment:
Looks good to me. The problem was that `make` didn't correctly return the exit code when the image build failed?
I assume that the makefile is still used in other scenarios (outside travis)?

Reply:
Yes, but where it matters what are the exit codes... And since I had issues convincing make to actually exit properly, I just replaced it with the simple practical thing that is actually doing,